Rohit/PostgresAssignment/controller/result.ts

```diff
@@ -0,0 +1,131 @@
+import { Context } from "vm";
+
+const res = require('../sql/result');
+
+interface resultinfo {
+    result_id: number;
```

Comment:
we dont need resultid, we can make classId, studentId, subjectId as primary key.